src/utils/subscriberBreaches.test.ts

```diff
@@ -80,12 +80,16 @@ const subscriber: SubscriberRow = {
       },
     },
   },
-  monthly_email_at: new Date("2022-08-07 14:22:00.000-05"),
+  monthly_email_at: "2022-08-07 14:22:00.000-05",
   monthly_email_optout: false,
   signup_language: "fr-CH, fr;q=0.9, en;q=0.8, de;q=0.7,*;q=0.5",
-  db_migration_1: undefined,
-  db_migration_2: undefined,
+  db_migration_1: null,
```

Comment:
I think these can be taken out now

db_migration_1
db_migration_2

Reply:
Removed these columns from the subscribers table in [2a7e588](https://github.com/mozilla/blurts-server/pull/4696/commits/2a7e58827df4e7ad108536226b148677bc67cbf1).